skaffold.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 Owkin, inc.
```

Comment:
Not sure if it's relevant, but could be moved to the `ci/` folder (it would make sense if this yaml file is only used for the ci) ?
Same question for the charts and docker folders?

Reply:
- the docker/chart/skaffold for `substra-test` can potentially be used for other contexts (not CI)
- the docker/chart for `substra-test=stack` is designed for the CI only

I would leave the structure as it is now for simplicity and consistenct with other repos, although I don't have super stromg feelings one way or another.

Reply:
I don't have a strong opinion on this also, so as you said let's keep this as it is. Thanks for the answer.